Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2584,3 +2584,121 @@ Function Get-AVSVSANClusterUNMAPTRIM {
             Get-Cluster | Get-VsanClusterConfiguration | Select-Object Name, GuestTrimUnmap
         }
 }
+
+
+Function Set-AVSVSANClusterEncryptionInTransit {
+    <#
+    .DESCRIPTION
+        This function enables vSAN Encryption in transit on the cluster defined by the -Name parameter.
+    .PARAMETER Name
+        Name of Clusters as defined in vCenter.  Valid values are blank or a comma separated list of cluster names.
+        Set-AVSVSANClusterEncryptionInTransit -Name Cluster-1,Cluster-2,Cluster-3
+        Enables vSAN Encryption in transit on Clusters-1,2,3
+        Set-AVSVSANClusterEncryptionInTransit -Enable:True
+        Enables vSAN Encryption in transit on all Clusters
```

Comment:
We don't have a way to show multi-line parameter description as a reasonably looking tooltip. Need to keep it singleline/short and move the examples to .EXAMPLE section (although we don't have a way to show that either).

Reply:
Any way to add something like to the PR validation check?  Also maybe add to scripting guidelines?

Reply:
Updated all to single line.